<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qt_AstarClass</class>
 <widget class="QMainWindow" name="Qt_AstarClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1108</width>
    <height>829</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt_Astar</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="CentFrom" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>701</width>
        <height>701</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>701</width>
        <height>701</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_selectstart">
          <property name="text">
           <string>选择起点</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_selectwall">
          <property name="text">
           <string>选择障碍</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_selectend">
          <property name="text">
           <string>选择终点</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_start">
          <property name="text">
           <string>开始</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_reset">
        <property name="text">
         <string>重置</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_time">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;00:00:00&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1108</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CentFrom</class>
   <extends>QWidget</extends>
   <header location="global">centfrom.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qt_astar.qrc"/>
 </resources>
 <connections/>
</ui>
